website/docs/r/compute_backend_bucket.html.markdown

```diff
@@ -34,28 +57,42 @@ resource "google_storage_bucket" "image_bucket" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the backend bucket.
+* `bucket_name` -
+  (Required)
+  Cloud Storage bucket name.
+* `name` -
+  (Required)
+  Name of the resource. Provided by the client when the resource is
+created. The name must be 1-63 characters long, and comply with
+RFC1035.  Specifically, the name must be 1-63 characters long and
+match the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means
+the first character must be a lowercase letter, and all following
+characters must be a dash, lowercase letter, or digit, except the
+last character, which cannot be a dash.
 
-* `bucket_name` - (Required) The name of the Google Cloud Storage bucket to be used as a backend
-    bucket.
 
 - - -
 
-* `description` - (Optional) The textual description for the backend bucket.
-
-* `enable_cdn` - (Optional) Whether or not to enable the Cloud CDN on the backend bucket.
+* `description` -
+  (Optional)
+  An optional textual description of the resource; provided by the
+client when the resource is created.
+* `enable_cdn` -
+  (Optional)
+  If true, enable Cloud CDN for this BackendBucket.
+* `project` (Optional) The project in which the resource belongs.
+    If it is not provided, the provider project is used.
 
-* `project` - (Optional) The ID of the project in which the resource belongs. If it is not provided, the
```

Comment:
Somebody felt strongly about saying "project ID" - do we want to keep that?

Reply:
Done. I followed the format introduced in #672